GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -57,7 +57,10 @@ public static void Register(GVFSPlatform platform)
 
         public abstract Dictionary<string, string> GetPhysicalDiskInfo(string path, bool sizeStatsOnly);
 
-        public abstract bool IsConsoleOutputRedirectedToFile();
+        public abstract bool IsConsoleOutputRedirectedToFile();
+
+        public abstract bool KillProcessTree(int processId);
```

Comment:
Make this `void` instead of `bool`? It doesn't look like any callers are checking the return value.

(If you leave it `bool` consider renaming to `TryKillProcessTree`)